tests/binary.rs

```diff
@@ -0,0 +1,93 @@
+#[cfg(feature = "binaries")]
+mod binary {
+  use assert_cmd::Command;
+  use rand::distributions::Alphanumeric;
+  use rand::{thread_rng, Rng};
+  use std::env::temp_dir;
+  use std::fs::File;
+  use std::io::Read;
+  use std::path::PathBuf;
+
+  fn get_y4m_input() -> Vec<u8> {
+    let mut input = File::open(&format!(
+      "{}/tests/small_input.y4m",
+      env!("CARGO_MANIFEST_DIR")
+    ))
+    .unwrap();
+    let mut data = Vec::new();
+    input.read_to_end(&mut data).unwrap();
+    data
+  }
+
+  fn get_tempfile_path(extension: &str) -> PathBuf {
```

Comment:
You may use [tempfile](https://docs.rs/tempfile/3.1.0/tempfile/index.html) maybe?

Reply:
I looked into that, but tempfile doesn't have an option to just generate a filename--it also creates the file. For this, we just want a filename in the temp directory that rav1e can write to.